src/stories/Library/links/LinkButton.tsx

```diff
@@ -0,0 +1,47 @@
+import clsx from "clsx";
+import { ButtonIcon } from "../Buttons/button/ButtonIcon";
+import {
+  ButtonSize,
+  ButtonType,
+  ButtonVariant,
+  getSize,
+  getVariant,
+} from "../Buttons/button/helper";
```

Comment:
I appreciate the intent of moving these functions to helpers but now we are at a place where we import helpers across components. This is something that we normally want to avoid. 

To me this is a sign that we should reconsider whether this component should be part of the `Link` group or the `Buttons` group.